src/src/WebServer/NotificationPage.cpp

```diff
@@ -207,8 +207,8 @@ void handle_notifications() {
           addFormTextBox(F("Domain"), F("domain"), NotificationSettings.Domain, sizeof(NotificationSettings.Domain) - 1);
           addFormTextBox(F("Server"), F("server"), NotificationSettings.Server, sizeof(NotificationSettings.Server) - 1);
           addFormNumericBox(F("Port"), F("port"), NotificationSettings.Port, 1, 65535, F("NOTE: SSL/TLS servers NOT supported!"));
-          if (NotificationSettings.Timeout<NPLUGIN_001_MIN_TM || NotificationSettings.Timeout>NPLUGIN_001_MAX_TM) {NotificationSettings.Timeout=NPLUGIN_001_DEF_TM;}
-          addFormNumericBox(F("Timeout (Secs)"), F("timeout"), NotificationSettings.Timeout, NPLUGIN_001_MIN_TM, NPLUGIN_001_MAX_TM, F("Maximum Server Response Time)"));
+          if (NotificationSettings.Timeout<NPLUGIN_001_MIN_TM/1000 || NotificationSettings.Timeout>NPLUGIN_001_MAX_TM/1000) {NotificationSettings.Timeout=NPLUGIN_001_DEF_TM/1000;}
+          addFormNumericBox(F("Timeout (Secs)"), F("timeout"), NotificationSettings.Timeout, NPLUGIN_001_MIN_TM/1000, NPLUGIN_001_MAX_TM/1000, F("Maximum Server Response Time)"));
```

Comment:
Just keep the timeout everywhere as msec, so you don't need to divide or multiply by 1000.
Removes a lot of complexity from the code.

You can add the unit right of the input field:
```c++
addFormNumericBox(F("Timeout"), F("timeout"), NotificationSettings.Timeout, NPLUGIN_001_MIN_TM, NPLUGIN_001_MAX_TM, F("Maximum Server Response Time)"));
addUnit(F("ms"));
```